python/export_util.py

```diff
@@ -76,24 +76,34 @@ def convert_to_isoformat(
         return property
 
 
-def get_graph(ctx: mgp.ProcCtx) -> List[Union[Node, Relationship]]:
+def get_graph(
+    ctx: mgp.ProcCtx, write_properties: bool
+) -> List[Union[Node, Relationship]]:
     nodes = list()
     relationships = list()
 
     for vertex in ctx.graph.vertices:
         labels = [label.name for label in vertex.labels]
-        properties = {
-            key: convert_to_isoformat(vertex.properties.get(key))
-            for key in vertex.properties.keys()
-        }
+        properties = (
+            {
+                key: convert_to_isoformat(vertex.properties.get(key))
+                for key in vertex.properties.keys()
+            }
+            if write_properties
+            else {}
+        )
```

Comment:
I think this can be part of function since you are duplicating this below on relationships